entrypoint.sh

```diff
@@ -70,6 +70,6 @@ else
         echo "FATAL: You must upgrade to using actions inputs instead of args: to push a single branch" > /dev/stderr
         exit 1
     else
-        eval git push -u ${GIT_PUSH_ARGS} mirror
+        eval git push -u ${GIT_PUSH_ARGS} mirror "${GITHUB_REF}"
```

Comment:
I seem to be having an issue when using this action in a PR where GITHUB_REF is being populated incorrectly with a `pull_request` trigger https://stackoverflow.com/questions/58033366/how-to-get-the-current-branch-within-github-actions.

My suggestion would be to let the user pass this in as an optional parameter, as that way they could use something like `BRANCH_NAME: ${{ github.head_ref || github.ref_name }}` to avoid such issues.

I tried a call to `- run: echo "GITHUB_REF=${{ github.head_ref }}" >> $GITHUB_ENV` before running this action, but this variable seems sticky and I was unable to work around it.

I ended up just reverting to `PUSH_ALL_REFS: "true"`

Reply:
Sure thing - I'll see if I can draft something up soon.

Reply:
Sorry for being late to the conversation here. I've created #33 to track this further.